2022.4.0 (#22253)
home-assistant/home-assistant.github.io: source/_posts/2022-04-06-release-20224.markdown

---
layout: post
title: "2022.4: Beta release notes"
description: "Beta release notes for Home Assistant Core 2022.4"
date: 2022-03-30 00:00:00
date_formatted: "April 06, 2022"
author: Franck Nijhof
author_twitter: frenck
comments: true
categories:
- Release-Notes
- Core
og_image: /images/blog/2022-04/social.png
---

<a href='/integrations/#version/2022.4'><img src='/images/blog/2022-04/social.png' style='border: 0;box-shadow: none;'></a>

> These are the beta release notes for Home Assistant Core 2022.4 (and is thus a

Reviewer: Remove this before merge.

great step forward in streamlining all the user experiences.

I could not fit more in these release notes, but there is more to tell!
Zack and Paulus promised to add a couple of great (and exclusive)
announcements to [the newsletter](/newsletter)! The newsletter has, by the way,
been renamed: _[Building the Open Home](/newsletter)_. Be sure to check it out!

I want to invite you to read the (long) release notes below. I'm sure there is
a lot in there you will like and love ❤️.

Let me know what you think the killer feature of this release is?

Enjoy the release!

../Frenck

- [Goodbye Lovelace, Hello Dashboards](#goodbye-lovelace-hello-dashboards)
- [Hide entities](#hide-entities)
- [Now available in the UI: Groups!](#now-available-in-the-ui-groups)
- [Switch as X](#switch-as-x)
- [More helpers now available via the UI](#more-helpers-now-available-via-the-ui)
- [Introducing update entities](#introducing-update-entities)
- [Backups for Home Assistant Core & Container installs](#backups-for-home-assistant-core--container-installs)
- [Testing your automation conditions](#testing-your-automation-conditions)
- [Zones now have a state!](#zones-now-have-a-state)

> - [Adjusting unit of measurements for sensors](#adjusting-unit-of-measurements-for-sensors)

Reviewer: ```suggestion
- [Adjusting unit of measurement for sensors](#adjusting-unit-of-measurement-for-sensors)
```

automatically!

<img class="no-shadow" src='/images/blog/2022-04/group-hide-members.png' alt='Screenshot showing how to hide group members'>

Your groups can be found under {% my helpers title="Configuration > Automations & Scripts > Helpers" %},
or, if you have updated to this release already, click the my button below to
add a new group immediately:

{% my config_flow_start badge domain="group" %}

## Switch as X

In Home Assistant’s world, a wall plug is a switch. And while that is correct
for a wallplug in general, those plugs (or relays) are often used with a light
fixture, a fan, garage door, or maybe your Christmas tree lights. 🎄

This release introduces the “Switch as X” helper, which lets you convert any
Home Assistant switch into a light, cover, fan, lock, or siren to match your
real-world use better.

Switch as X can be found under {% my helpers title="Configuration > Automations & Scripts > Helpers" %},
but wait! We made it easier: You can also change the switch itself when
editing the switch entity:

<div class='videoWrapper'>
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/FLH4SXMAQS4" frameborder="0" allowfullscreen></iframe>
</div>

> The video above shows the creating of the new light entity, which uses the

Reviewer: ```suggestion
The video above shows how to create a new light entity, which uses the
```

original switch. The original switch is now automatically hidden from most of

> the Home Assistant UI but is still available on this device page

Reviewer: ```suggestion
the Home Assistant UI but is still available on this device page.
```

If you have updated to this release already, click the my button below to
transform one of your switches right away:

{% my config_flow_start badge domain="switch_as_x" %}

## More helpers now available via the UI

The above Groups and Switch as X, are to be found in what we call: Helpers.

Helpers are integrations that expose a way for a user to provide input or
consume existing data in Home Assistant to provide a new entity.

> The latter category of helpers has very much extended in this release!

Reviewer: ```suggestion
The latter category of helpers has very much been extended in this release!
```

Not just groups and switch as X, but we have many more helper integrations
available in the UI now!

This release brings the following helpers to the UI:

- [Derivative]
- [Groups]
- [Integration - Riemann sum integral]
- [Min/Max]
- [Switch as X]
- [Threshold]
- [Times of day]
- [Utility Meter]

[Derivative]: /integrations/derivative
[Groups]: /integrations/group
[Integration - Riemann sum integral]: /integrations/integration
[Min/Max]: /integrations/min_max
[Switch as X]: /integrations/switch_as_x
[Threshold]: /integrations/threshold
[Times of Day]: /integrations/tod
[Utility Meter]: /integrations/utility_meter

## Introducing update entities

Say "hi!" to a brand new entity type: `update`.

Update entities can tell you if an update is available for your device and
service and, in some cases, allow you to install the update straight from
Home Assistant!

And the beautiful thing is: They show up in your configuration Dashboard, just
like a Home Assistant, Home Assistant OS, or add-on update. As a matter of fact,
those are now update entities too!

> Now updates are entities other integrations can provide them too! For example,

Reviewer: ```suggestion
Now updates are entities and other integrations can provide them too! For example,
```

Author: Hmm I think this should be a comma?

```suggestion
Now updates are entities, other integrations can provide them too! For example,
```

Reviewer 2: I think comma too 👍

</p>

These `update` entities can be provided by integrations and provide more
information about the update, like: Version information, links to release notes,
and the possibility to skip that specific version offered.

The blue number indicator on the configuration cog icon in your sidebar tells
you how many updates are pending.

The Home Assistant Supervisor (providing Core, OS, and add-on updates), WLED,
Pi-hole, and Synology DSM integrations have implemented these brand new
update entities in this release.

And because they are all entities, it means you can automate with them. For
example, you could send a notification when a new add-on update is available
for installation.

## Backups for Home Assistant Core & Container installs

If you are running Home Assistant OS or Supervised, you already have access to
Home Assistant's really helpful backup capabilities. However, this feature was
unavailable if you run Home Assistant Core in Python or use a Home Assistant
Container in Docker.

That all changes this release: Introducing a Backup integration for Home
Assistant Core and Container installation types!

<p class='img'>

> <img class="no-shadow" src='/images/blog/2022-04/backups.png' alt='Screenshot showing backup creating in progress on a Home Assistant Core installation type.'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2022-04/backups.png' alt='Screenshot showing backup creation in progress on a Home Assistant Core installation type.'>
```

Home Assistant Core and Container installations can now also back up their instance.
</p>

These backups can be downloaded and extracted if you want or need to restore one
of the previous backups. Also, these backups are compatible with Home Assistant
OS installations. So, if you plan on migrating installation types in the future,
that is now super easy!

You can find the backup dashboard in the configuration menu, or, if you have

> upgrade already, click the my button below to create a new backup right away:

Reviewer: ```suggestion
upgraded already, click the my button below to create a new backup right away:
```

{% my backup badge %}

## Testing your automation conditions

In our previous release we [added a feature][triggered] to see if the automation
you create triggers correctly. During the release party live stream,
we got questions: _"Can this be done for conditions too?"_; Sure we can!

<p class='img'>
<img class="no-shadow" src='/images/blog/2022-04/test-condition.gif' alt='Screen recording showing the new test button that allows you to test the condition'>
Every condition now has a little test button, to test if the condition passes right now.
</p>

Additionally, the “triggered” bar got an additional feature too! You can click

> on it to get a raw view of all trigger variable data if it shows. This data

Reviewer: ```suggestion
on it to get a raw view of all trigger variable data. This data
```

can be helpful in case you want to create an advanced condition or template
using that information.

> <img class="no-shadow" src='/images/blog/2022-04/triggered-variables.png' alt='Screenshot showing variable triggered data when the triggered notification is clicked'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2022-04/triggered-variables.png' alt='Screenshot showing variable trigger data when the triggered notification is clicked'>
```

[triggered]: /blog/2022/03/02/release-20223/#triggered

## Zones now have a state!

If you ever looked at the state of a `zone` entity, you would have noticed
it always said: _"Zoning"_. Not really helpful 🙃

So, to improve on that, we changed that behavior. The state of a `zone` entity
will now show the number of [persons] currently in that zone; that is more
helpful! Additionally, it means that a zone has history now too!

<img class="no-shadow" src='/images/blog/2022-04/zone-state.png' alt='Screenshot showing the new numeric state of our zones'>

This will greatly simplify automations and scripts that, for example, trigger
on: Nobody is home, everybody is home, someone left home (or any other zone 
for that matter). Or use it as a condition to prevent an automation
from running when nobody is home.

<p class='img'>
<img class="no-shadow" src='/images/blog/2022-04/zone-condition.png' alt='Screenshot showing how to use zone with in a numeric condition'>

> The condition ensure this automation will only trigger if someone is home.

Reviewer: ```suggestion
The condition ensures this automation will only trigger if someone is home.
```

</p>

[persons]: /integrations/person

> ## Adjusting unit of measurements for sensors

Reviewer: ```suggestion
## Adjusting unit of measurement for sensors
```

Got that one sensor measuring a temperature in Celsius, but you prefer it to be
in Fahrenheit? No problem! You can now change the unit of measurement for
temperature and pressure sensors straight from the UI.

> <img class="no-shadow" src='/images/blog/2022-04/adjust-unit-of-measurement.png' alt='Screenshot showing adjusting the unit for measurement for an temperature entity'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2022-04/adjust-unit-of-measurement.png' alt='Screenshot showing adjusting the unit for measurement for a temperature entity'>
```

## Adjusting long-term statistics

Long-term statistics are great! Until... something went wrong 😅

Unfortunately, things happen. For example, it registered a big spike, or your
solar or electricity meter provided a one-off rogue measurement. These events
impact your long-term statistics, which might forever have incorrect history.

We now have made available a little developer tool that allows you to
correct/adjust long-term historical data to solve these issues.

The new tool can be found at {% my developer_statistics title="Developer tools > Statistics" %},
and click the
<svg width="24" height="24" viewBox="0 0 24 24"><path d="M22,13V22H2V19L22,13M21.68,7.06L16.86,4.46L17.7,7.24L7.58,10.24C6.63,8.95 4.82,8.67 3.53,9.62C2.24,10.57 1.96,12.38 2.91,13.67C3.85,14.97 5.67,15.24 6.96,14.29C7.67,13.78 8.1,12.97 8.14,12.09L18.26,9.09L19.1,11.87L21.68,7.06Z" /></svg>
icon, next to the statistic you'd like to adjust. Search for the incorrect

> datapoint by date & time and adjust the value.

Reviewer: ```suggestion
The new tool can be found at {% my developer_statistics title="Developer tools > Statistics" %}.
Click the
<svg width="24" height="24" viewBox="0 0 24 24"><path d="M22,13V22H2V19L22,13M21.68,7.06L16.86,4.46L17.7,7.24L7.58,10.24C6.63,8.95 4.82,8.67 3.53,9.62C2.24,10.57 1.96,12.38 2.91,13.67C3.85,14.97 5.67,15.24 6.96,14.29C7.67,13.78 8.1,12.97 8.14,12.09L18.26,9.09L19.1,11.87L21.68,7.06Z" /></svg>
icon, next to the statistic you'd like to adjust, search for the incorrect
datapoint by date & time and adjust the value.
```

<img class="no-shadow" src='/images/blog/2022-04/adjust-statistics.png' alt='Screenshot showing adjusting the long-term statistic history value.'>

{% my developer_statistics badge %}

## Optimizing performance

This release brings significant performance optimizations to both Home
Assistant's backend and frontend UI.

### Frontend UI performance <!-- omit in toc -->

The communication layer between the Home Assistant frontend and its Core backend
has been greatly improved. It now sends over a lot less data and is heavily
optimized to send the least amount of data when first loading up the UI.

The UI will load faster since less data is transferred over the wire,
and there is less data for the web browser or app to process.

> Additionally, lessened data transmitted means Home Assistant will work better

Reviewer: ```suggestion
Additionally, transmitting less data means Home Assistant will work better
```

attributes instead of saving the whole set of state attributes.  

As data stored in the database before 2022.4 does not get updated,
the reduction in database size will occur over time with the recording
of new data and the purging of old data.

On top of that, most entity history graphs no longer need to fetch the
attributes from the backend, and as a result, they load much faster after
24 hours of running on 2022.4. This change is especially noticeable
for energy sensors.  

For a complete set of changes to the underlying database,
check out the [Data Science Portal].

[Data Science Portal]: https://data.home-assistant.io/docs/states#database

## New and updated selectors!

Selectors are UI inputs used throughout Home Assistant. It powers our new UI
helpers, automation editor, and provides UIs for Blueprints and scripts.

We added a whole bunch of [new selectors][selectors] that are usable in your
Blueprints, and extended the options for many [existing selectors][selectors].

For example, we now have a location selector that lets you pick coordinates
from a map. An RGB color and color temperature selector, an icon, theme, time,

> date, datetime selector; The entity, device, and area select can now be

Reviewer: ```suggestion
date, datetime selector; The entity, device, and area selectors can now be
```

For the more advanced uses cases, this release provides: variables on trigger.
It is exactly what it says on the can: The possibility of setting variables
on triggers that become available when that specific trigger triggers.

These variables can be helpful to pass down specific information based on the
trigger that fired. The automation example below shows how a message can be
passed to the notify service based on its triggering motion sensor.

{% raw %}

```yaml
automation:
  trigger:
    - platform: state
      entity_id: binary_sensor.motion_garage
      to: "on"
      variables:
        trigger_message: "Motion detected in the garage! 🚘"
    - platform: state
      entity_id: binary_sensor.motion_garden
      to: "on"
      variables:
        trigger_message: "Motion detected in the garden! 🌷"
  action:
    - service: notify.iphone
      data:
        message: "{{ trigger_message }}"
```

{% endraw %}

Oh! And yes: these variables can use templates too!

## Other noteworthy changes

There is much more juice in this release; here are some of the other
noteworthy changes this release:

- [@balloob] took a deep dive into the world of media players and made sure

>   many more of them now work the media browser and play media action!

Reviewer: ```suggestion
  many more of them now work with the media browser and play media action!
```

  If you use: [Bluesound], [Denon HEOS], [forked-daapd], [Linn / OpenHome],
  [Logitech Squeezebox], [Music Player Daemon (MPD)], [Panasonic Viera],
  [UniFi Protect], or [Yamaha MusicCast], you can now use the media browser!
- Want to update the appearance of a binary sensor? You can now do that!
  When you edit the entity in the UI, you can select how it would show up. No
  need to customize `device_class` in YAML anymore. Thanks, [@zsarnett]!
- The [Material Design Icons] have been updated! Check out all 
  [these beautiful new icons] we can use now! Thanks, [@goyney]!
- Our analytics integration now sends if you are using SSL certificates 

>   in Home Assistant (yes or no). Of course,

Reviewer: ```suggestion
- Our analytics integration now includes data on using SSL certificates 
  in Home Assistant (yes or no) in the analytics. Of course,
```

has been deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration
after upgrading.

([@rappenze] - [#65203]) ([documentation](/integrations/fibaro))

[@rappenze]: https://github.com/rappenze
[#65203]: https://github.com/home-assistant/core/pull/68702

{% enddetails %}

{% details "File Size" %}

The File Size integration migrated to configuration
via the UI. Configuring File Size via YAML configuration
has been deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration
after upgrading.

([@gjohansson-ST] - [#67668]) ([documentation](/integrations/filesize))

[@gjohansson-ST]: https://github.com/gjohansson-ST
[#67668]: https://github.com/home-assistant/core/pull/67668

---

Additional attributes for the File Size sensors has been extracted into
their own sensors which are disabled by default.

If you were previously using any of these attributes your automations,

> scripts etc. needs to be updated to use the new sensors.

Reviewer: ```suggestion
scripts etc. need to be updated to use the new sensors.
```

[#68678]: https://github.com/home-assistant/core/pull/68678

{% enddetails %}

{% details "MySensors" %}

The MySensors integration migrated to configuration via the UI a year ago.
Configuring MySensors via YAML configuration has been deprecated and
will be removed in a future Home Assistant release.

Your existing YAML configuration has been automatically imported since a
long time; and thus can be safely removed from your YAML configuration.

([@MartinHjelmare] - [#68504]) ([documentation](/integrations/mysensors))

[@MartinHjelmare]: https://github.com/MartinHjelmare
[#68504]: https://github.com/home-assistant/core/pull/68504

{% enddetails %}

{% details "Nanoleaf" %}

The previously deprecated YAML configuration of the Nanoleaf
integration has been removed.

Nanoleaf is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@balloob] - [#69004]) ([documentation](/integrations/nanoleaf))

[@balloob]: https://github.com/balloob
[#69004]: https://github.com/home-assistant/core/pull/69004

{% enddetails %}

{% details "Nest" %}

> Legacy Works with Net API deprecation is put on hold until issues with

Reviewer: ```suggestion
Legacy Works with Nest API deprecation is put on hold until issues with
```

as `sensor.opentherm_outdoor_air_temperature`.

The use of the zipcode-based `outdoor_temperature` for other devices has
not changed.

([@CoMPaTech] - [#68303]) ([documentation](/integrations/plugwise))

[@CoMPaTech]: https://github.com/CoMPaTech
[#68255]: https://github.com/home-assistant/core/pull/68255

{% enddetails %}

{% details "PVOutput" %}

The previously deprecated YAML configuration of the PVOutput
integration has been removed.

PVOutput is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@frenck] - [#67162]) ([documentation](/integrations/pvoutput))

[@frenck]: https://github.com/frenck
[#67162]: https://github.com/home-assistant/core/pull/67162

---

All, previously deprecated, additional sensor attributes of the PVOutput sensor,
have now been removed. All of these attributes now have their own dedicated
sensor.

This applies to the following PVOutput attributes:

- `energy_generation`
- `power_generation`
- `energy_consumption`
- `power_consumption`
- `efficiency`
- `temperature`
- `voltage`

> If you relied on these in your automation or scripts, you need to adopt those

Reviewer: ```suggestion
If you relied on these in your automations or scripts, you need to adopt those
```

when using custom queries or doing data analytics on your database.

---

Attributes are now stored in a `state_attributes` table, storing the
same set of attributes once (many to one relationship).

Attributes represent roughly 21% of the database size. (28% if you exclude
statistics). On a few of the analyzed production instances, attributes
ranged from 82-88% duplicates of another set of attributes. Expected is
that this change will reduce the database size by roughly between 13-16%
on average.

([@bdraco] - [#68224]) ([documentation](/integrations/recorder))

[@bdraco]: https://github.com/bdraco
[#68224]: https://github.com/home-assistant/core/pull/68224

---

The `domain` column has been removed from the `states` table. The domain can
be derived from the `entity_id`.

([@bdraco] - [#68404]) ([documentation](/integrations/recorder))

[@bdraco]: https://github.com/bdraco
[#68404]: https://github.com/home-assistant/core/pull/68404

---

The attributes for the `sun.sun` entity are no longer recorded in the database.

As historical data can be derived from the `astral` library without the
need to fetch it the database, it didn't seem like it needs to be copied

> there as well since it frequently generates new attributes rows in the database.

Reviewer: ```suggestion
As historical data can be derived from the `astral` library without the
need to fetch it from the database, it was decided to remove the sun attributes
from the database since it frequently generates new attribute rows in the database.
```

- Select: `options`.
- Siren: `available_tones`.
- Vacuum: `fan_speed_list`.
- Weather: `forecast`.

([@bdraco] - [#68404] [#69155] [#69156] [#69158] [#69159] [#69165] [#69192] [#69193] [#69194] [#69195] [#69196] [#69199])

([@frenck] - [#69205] [#69209])

[@bdraco]: https://github.com/bdraco
[#68824]: https://github.com/home-assistant/core/pull/68824
[#69155]: https://github.com/home-assistant/core/pull/69155
[#69158]: https://github.com/home-assistant/core/pull/69158
[#69192]: https://github.com/home-assistant/core/pull/69192
[#69156]: https://github.com/home-assistant/core/pull/69156
[#69159]: https://github.com/home-assistant/core/pull/69159
[#69165]: https://github.com/home-assistant/core/pull/69165
[#69193]: https://github.com/home-assistant/core/pull/69193
[#69194]: https://github.com/home-assistant/core/pull/69194
[#69195]: https://github.com/home-assistant/core/pull/69195
[#69196]: https://github.com/home-assistant/core/pull/69196
[#69199]: https://github.com/home-assistant/core/pull/69199
[#69205]: https://github.com/home-assistant/core/pull/69205
[#69209]: https://github.com/home-assistant/core/pull/69209

{% enddetails %}

{% details "RFXCOM RFXtrx" %}

> Support for signal repetition for devices have been removed. RFXCOM internally

Reviewer: ```suggestion
Support for signal repetition for devices has been removed. RFXCOM internally
```

> repeat packets as needed per protocol.

Reviewer: ```suggestion
repeats packets as needed per protocol.
```

([@elupus] - [#67675]) ([documentation](/integrations/rfxtrx))

[@elupus]: https://github.com/elupus
[#67675]: https://github.com/home-assistant/core/pull/67675

{% enddetails %}

{% details "Sensibo" %}

Sensibo Air devices with an active room sensor (as the main measurement sensor)
will now retrieve the temperature and humidity from the room sensor.

([@gjohansson-ST] - [#66949]) ([documentation](/integrations/sensibo))

[@gjohansson-ST]: https://github.com/gjohansson-ST
[#66949]: https://github.com/home-assistant/core/pull/66949

---

> The previously deprecated YAML configuration of the Version

Reviewer: ```suggestion
The previously deprecated YAML configuration of the Sensibo
```

integration has been removed.

> Version is now configured via the UI, any existing YAML

Reviewer: ```suggestion
Sensibo is now configured via the UI, any existing YAML
```

[@balloob]: https://github.com/balloob
[#69002]: https://github.com/home-assistant/core/pull/69002

{% enddetails %}

{% details "Synology DSM" %}

The binary sensor entity that would show if there was an update available for
the Synology DSM firmware has been deprecated and will be removed in
Home Assistant Core 2022.6.

The Synology DSM integration now provides an `update` entity to replace
the deprecated entity.

([@mib1185] - [#68664]) ([documentation](/integrations/synology_dsm))

[@mib1185]: https://github.com/mib1185
[#68664]: https://github.com/home-assistant/core/pull/68664

{% enddetails %}

{% details "Tankerkoenig" %}

The Tankerkoenig integration migrated to configuration
via the UI. Configuring Tankerkoenig via YAML configuration
has been deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration
after upgrading.

([@mib1185] - [#68386]) ([documentation](/integrations/tankerkoenig))

[@mib1185]: https://github.com/mib1185
[#68386]: https://github.com/home-assistant/core/pull/68386

---

> The `is_open` attribute is removed from all entities and now exposed as own

Reviewer: ```suggestion
The `is_open` attribute is removed from all entities and now exposed as its own
```

sensor.

> If you have automations or script relying on this attribute, you will need

Reviewer: ```suggestion
If you have automations or scripts relying on this attribute, you will need
```

has been deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration
after upgrading.

([@gjohansson-ST] - [#65182]) ([documentation](/integrations/trafikverket_train))

[@gjohansson-ST]: https://github.com/gjohansson-ST
[#65182]: https://github.com/home-assistant/core/pull/65182

{% enddetails %}

{% details "Trafikverket Weather Station" %}

The previously deprecated YAML configuration of the Trafikverket Weather Station
integration has been removed.

Trafikverket Weather Station is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@tkdrob] - [#68336]) ([documentation](/integrations/trafikverket_weatherstation))

[@tkdrob]: https://github.com/tkdrob
[#68336]: https://github.com/home-assistant/core/pull/68336

---

- The attribute for measure time is now a proper iso formatted string.
- The attribute for available has been removed and is now setting the entity

>   available or not.

Reviewer: ```suggestion
- The attribute for available has been removed
  and the entity instead sets its state to unavailable when not available.
```

([@dgomes] - [#55690]) ([documentation](/integrations/utility_meter))

[@dgomes]: https://github.com/dgomes
[#55690]: https://github.com/home-assistant/core/pull/55690

{% enddetails %}

{% details "Version" %}

The previously deprecated YAML configuration of the Version
integration has been removed.

Version is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@balloob] - [#69010]) ([documentation](/integrations/version))

[@balloob]: https://github.com/balloob
[#69010]: https://github.com/home-assistant/core/pull/69010

{% enddetails %}

{% details "Whois" %}

The previously deprecated YAML configuration of the Whois
integration has been removed.

Whois is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@frenck] - [#67163]) ([documentation](/integrations/whois))

[@frenck]: https://github.com/frenck
[#67163]: https://github.com/home-assistant/core/pull/67163

{% enddetails %}

{% details "WLED" %}

> The firm update `binary_sensor` and `button` entities are deprecated and

Reviewer: ```suggestion
The firmware update `binary_sensor` and `button` entities are deprecated and
```

integration has been removed.

Yale Smart Alarm is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@frenck] - [#69025]) ([documentation](/integrations/yale_smart_alarm))

[@frenck]: https://github.com/frenck
[#69025]: https://github.com/home-assistant/core/pull/69025

{% enddetails %}

{% details "Yamaha Music Cast" %}

The previously deprecated YAML configuration of the Yamaha Music Cast
integration has been removed.

Yamaha Music Cast is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@frenck] - [#69033]) ([documentation](/integrations/yamaha_musiccast))

[@frenck]: https://github.com/frenck
[#69033]: https://github.com/home-assistant/core/pull/69033

{% enddetails %}

{% details "Z-Wave JS" %}

The Z-Wave JS siren entities will now return `unknown` as a state if the
corresponding driver value is missing.

Previously the state defaulted to `off` in these cases. You might need to

> consider thi in automations and scripts that rely on a Z-Wave JS siren

Reviewer: ```suggestion
consider this in automations and scripts that rely on a Z-Wave JS siren
```
